wiki/Help_Centre/en.md

```diff
@@ -270,17 +302,59 @@ See [this topic](https://answers.microsoft.com/en-us/windows/forum/windows_vista
+4. Make sure `Display beatmaps from` is set to 0 stars, and that `up to` is set to 10+ stars.
+
+![](img/beatmapdiffdisp.png "The Diplay Beatmaps setting set to the correct values")
+
+If none of these methods work, as a last resort you can try forcing your beatmap database to be recreated. **Note that this will mark all your maps as "unplayed", so searching and grouping maps by fields that rely on date played will no longer work.**
+
+To safely force your beatmap database to be recreated, try the following steps:
+
+1. Open osu!.
+2. Enter the `Options` menu or hit `Control` + `O`.
+3. Click `Open osu! folder`.
+4. Close osu!.
+5. Find the `osu!.db` file located in the folder you just opened.
+6. Right-click on it, then click `Rename`.
+7. Rename it to anything. It does not matter what you rename it to, as long as it is not "osu!". Then press `Enter`.
+8. Relaunch osu!.
+
+#### My song list keeps scrolling!
+
+**This is usually due to a faulty input device connected to your computer. Try unplugging any controllers or joysticks.**
+
+Applications which emulate or change keybindings such as *Xpadder* or *X-Mouse Button Control* may also cause these issues. Try disabling them if you use them for other games.
+
+This can also happen due to issues with the numpad keys, as they can be used to scroll the song selection list. Pressing the `NumLock` key to disable the numpad and then pressing `9`, `8`, `3`, or `2` on the numpad will fix the issue.
+
+You may also wish to check for any jammed or damaged buttons on your peripherals.
+
+#### I downloaded a Beatmap Pack, but osu! is always trying to "repair" the file!
+
+**You will need to extract the pack into your Songs directory.**
+
+Most beatmap packs come in a `.rar` file format, which means they are a compressed archive of beatmap files. You will need to extract them using your utility of choice (we recommend [7-Zip](https://7-zip.org "7zip")) to your Songs directory first.
+
+Once all of the `.osz` files within the pack are in the Songs directory, hitting `F5` at the song selection menu will refresh the game's beatmap cache and load your new songs into the game.
+
 ### Gameplay
 
 #### My combo counter, score display or accuracy display is missing!
 
-**Pressing `Shift` + `Tab` by default will toggle the in-game HUD and allow you to see these elements again.**
+**Pressing `Shift` + `Tab` while playing will toggle the in-game HUD and allow you to see these elements again.**
 
-If you have changed the keybinding that toggles the scoreboard, this will not work. You can check which key it is set to in `Options` -> `Change keyboard bindings` -> `In-Game` -> `Toggle Scoreboard`.
+If you have changed the keybinding that toggles the scoreboard, this will not work. You can check which key it is set to in `Options` -> `Change keyboard bindings` -> `In-Game` -> `Toggle Scoreboard`. Then you can substitute that key in, in place of `Tab`. For example `Shift` + `V`.
```

Comment:
choppy sentences read poorly

```suggestion
If you have changed the keybinding that toggles the scoreboard, this will not work. You can check which combination it is set to in `Options` -> `Change keyboard bindings` -> `In-Game` -> `Toggle Scoreboard`. Then you can substitute the key in place of `Tab` (for example, to make it `Shift` + `V`).
```